tap_clickup/streams.py

```diff
@@ -4,6 +4,50 @@
 import requests
 from singer_sdk.helpers.jsonpath import extract_jsonpath
 from tap_clickup.client import ClickUpStream
+import yaml #added to read information specifying spaces and workspaces
```

Comment:
All of this needs to go away. Please go through the getting started for Meltano, and read through how singer works. 

Adding extra fields to a `config` block added here https://github.com/AutoIDM/tap-clickup/blob/main/meltano.yml#L6 will pass all of the configs via the config CLI argument (via a json file) to the tap. This is all handled by the config options you altered in `tap.py` below